src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -972,109 +972,277 @@ bool java_bytecode_convert_methodt::class_needs_clinit(
   return false;
 }
 
-/// Create a ::clinit_wrapper the first time a static initializer might be
-/// called. The wrapper method checks whether static init has already taken
-/// place, calls the actual <clinit> method if not, and initializes super-
-/// classes and interfaces.
-/// \param classname: Class name
+/// This function retrieves from or creates into the symbol table a wrapper
```

Comment:
It might be worth factoring out the clinit generation into a separate .cpp file in order to reduce the size of this monster.cpp.

Reply:
agreed, will do this.

Reply:
Hang on! This is already done in the PR this is racing against :)